pkg/routeagent_driver/handlers/ovn/connection.go

```diff
@@ -293,3 +338,22 @@ func FindPod(ctx context.Context, k8sClientset clientset.Interface, labelSelecto
 
 	return &podsList.Items[0], nil
 }
+
+//nolint:nilnil // Intentional as the purpose is to find.
+func FindOpenshiftNetwork(ctx context.Context, dynamicClient dynamic.Interface) (*unstructured.Unstructured, error) {
+	networkClient := dynamicClient.Resource(
+		schema.GroupVersionResource{
+			Group:    "config.openshift.io",
+			Version:  "v1",
+			Resource: "networks",
+		},
+	)
+
+	networks, err := networkClient.Get(ctx, "cluster", metav1.GetOptions{})
```

Comment:
Just to confirm, there is no need to add any additional RBAC right?

Reply:
Good point, 
I think that route-agent OCP RBAC Should be updated to support Network CRD